packages/config/src/env/main.js

```diff
@@ -78,11 +79,17 @@ const getGeneralEnv = async function ({
   context,
 }) {
   const gitEnv = await getGitEnv(buildDir, branch)
+
+  const DEPLOY_ID = deployId || 0
+  const BUILD_ID = buildId || 0
+
   return removeFalsy({
     SITE_ID: id,
     SITE_NAME: name,
```

Comment:
We should add a default value to `SITE_NAME` since this is now used in `DEPLOY_URL` and `DEPLOY_PRIME_URL`. Also, this is probably a good idea generally speaking.

What about something like this?

```js
const getGeneralEnv = async function ({
  siteInfo: { 
    id, 
    name = 'site-name', 
    url, 
    build_settings: { repo_url: REPOSITORY_URL } = {} },
```